NEWS

```diff
@@ -2,23 +2,207 @@ systemd System and Service Manager
 
 CHANGES WITH 231:
 
-        * When using systemd's default tmp.mount for /tmp, this will now be
-          mounted with the "nosuid" and "nodev" options. This avoids
-          privilege escalation attacks that put traps and exploits into /tmp.
-          However, this might cause some problems if you e. g. put container
+        * In service units the various ExecXYZ= settings have been extended
+          with an additional special character as first argument of the
+          assigned value: if the character '!' is used the specified command
+          line it will be run with full privileges, regardless of User=,
+          Group=, CapabilityBoundingSet= and similar options. The effect is
+          similar to the existing PermissionsStartOnly= option, but allows
+          configuration of this concept for each executed command line
+          independently.
```

Comment:
Hm, tbh I find it a bit confusing that '!' in the context of Condition means negate, and here it get's a different meaning.
Would it be possible to pick a different character?